NativeApp/src/components/User/container.js

```diff
@@ -47,6 +48,23 @@ export default Container => class extends Component {
     this.sub.remove();
   }
 
+  getDays = (events, description) => {
+    let totalDays = 0;
+    events.forEach((event) => {
+      if (!event.halfDay) {
+        totalDays += event.eventStatus.description === description
+          ? getDuration(event.start, event.end) : 0;
+      }
+
+      if (event.halfDay) {
```

Comment:
This can just be the `else` for the above `if`

Reply:
cool, just made the change :)